backend/gke/tfsetup.sh

```diff
@@ -28,6 +28,7 @@ gcp_dns_sa_key = "$GKE_DNSCRED_JSON"
 cluster_labels = {
     catapult-clustername = "$GKE_CLUSTER_NAME",
     owner = "${OWNER}"
```

Comment:
I suppose we need to be more intelligent here when terminating the owner object with a comma, when EXTRA_LABELS is set up.
Maybe it pays off to incrementally build a json with jq or similar.

Reply:
Addressed in 7127ab4176f026be59507c2d6d2f4d7e5a81568c — deployed a cluster and got the extra label.  Did this via `yq`, since we already had it as a dependency.